AISKU/rollup.config.js

```diff
@@ -3,7 +3,7 @@ import { uglify } from "@microsoft/applicationinsights-rollup-plugin-uglify3-js"
 import replace from "@rollup/plugin-replace";
 import cleanup from "rollup-plugin-cleanup";
 import dynamicRemove from "@microsoft/dynamicproto-js/tools/rollup/node/removedynamic";
-import { es3Poly, es3Check, importCheck } from "@microsoft/applicationinsights-rollup-es3";
```

Comment:
Actually, I should replace these with the new es5 versions -- I removed these before I decided to branch the es3 -> es5.
While there are no current automatic polyfills there are some checks that should still occurr.